pipeline/dbt/models/intermediate/int__union_adresses__enhanced.sql

```diff
@@ -17,27 +19,83 @@ valid_adresses AS (
             longitude,
             source
         ) AS errors ON TRUE
+        -- noqa: enable=references.qualification
     WHERE errors.field IS NULL
 ),
 
 geocoded_results AS (
     SELECT * FROM {{ ref('int_extra__geocoded_results') }}
 ),
 
-final AS (
+geocoded_addresses AS (
+
+    SELECT
+        valid_adresses._di_surrogate_id,
+        valid_adresses.id,
+        valid_adresses.source,
+        valid_adresses.complement_adresse,
+        geocoded_results.result_name     AS adresse,
```

Comment:
Quelle différence entre `geocoded_results.result_name` et `geocoded_results.result_label` ?

Reply:
```
result_label       | 29 Rue Marsoulan 75012 Paris
result_name        | 29 Rue Marsoulan
```

Reply:
got it